src/traversals/bfs.jl

```diff
@@ -189,12 +189,15 @@ function is_bipartite(g::SimpleGraph, s::Int)
     return visitor.is_bipartite
 end
 
-function is_bipartite(g::SimpleGraph)
-    nvg = nv(g)
-    for v in vertices(g)
-        if !is_bipartite(g, v)
-            return false
+if VERSION >= v"0.4.0-dev+6371"
+    is_bipartite(g::SimpleGraph) = all([is_bipartite(g,v) for v in vertices(g)])
```

Comment:
The annoying part here is this creates a temporary array and `is_bipartite` is now always called for every vertex. :s